app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -158,12 +158,12 @@ public String getApiKey() {
     public String getUserAgent() {
         try {
             return ZulipApp.USER_AGENT
-                    + "/"
+                    + "-"
```

Comment:
This should stay as a `/`.  User agents are usually in the form `software/version`